src/main/java/cms/gov/madie/measure/config/UpdateQdmMeasureGroupStratificationsChangeUnit.java

```diff
@@ -56,7 +56,8 @@ public void updateQdmMeasureGroupStratifications(MeasureRepository measureReposi
                                                           s.getDescription(),
                                                           s.getCqlDefinition(),
                                                           s.getAssociation(),
-                                                          s.getAssociations()))
+                                                          s.getAssociations(),
+                                                          s.getDisplayId()))
```

Comment:
this change log has already been executed in prod. is this just to avoid some compilation errors?

Reply:
Yes, The constructor is expecting the displayId, We do not have variable constructors

Reply:
@adongare Yeah just to avoid the compilation error.